include/aspect/gravity_model/radial.h

```diff
@@ -38,7 +38,7 @@ namespace aspect
      * @ingroup GravityModels
      */
     template <int dim>
-    class RadialConstant : public Interface<dim>
+    class RadialConstant : public Interface<dim>, public virtual SimulatorAccess<dim>
```

Comment:
Is there a reason to use virtual inheritance instead of simply `public SimulatorAccess<dim>`? If not, please remove the `virtual` (also in the two other classes).